api/src/pcapi/core/external/subcategory_suggestion_backends/subcategory_suggestion.py

```diff
@@ -43,9 +44,20 @@ def get_most_probable_subcategories(
         }
         response = requests.post(url, headers=headers, json=data, timeout=SUBCATEGORY_SUGGESTION_TIMEOUT_SECONDS)
         if response.status_code in {401, 403}:
-            logger.exception(
+            # FIXME (ogeber, 2024-12-12) this is to investigate the source of 403/401 errors from compliance,
+            # once the issue has been found, we can remove this complexe log
+            try:
+                response_json = response.json()
+            except JSONDecodeError:
+                response_json = None
```

Comment:
J'aurais fait plus simplement :
```suggestion
            try:
                response_data = response.json()
            except JSONDecodeError:
                response_data = response.text
```
puis `"response": response_data` en-dessous.
Mais c'est un détail :)

Reply:
c'était un peu "vite fait mal fait", c'est mieux comme ça en effet merci